src/base/stl_helpers/ostream_helpers.test.cc

```diff
@@ -90,3 +90,29 @@ TEST(OstreamHelpers, printer) {
     os2 << gem5::stl_helpers::Printer(hello);
     EXPECT_EQ(os2.str(), "[ H, e, l, l, o, ]");
 }
+
+
+TEST(OstreamHelpers, pointers) {
+    auto helpedRepresentation = [](const auto& val) {
```

Comment:
Nit: this should be lower case instead of lower camel case being a local variable